makecrx.sh

```diff
@@ -38,7 +38,10 @@ echo "Building chrome version" $VERSION
 [ -d pkg ] || mkdir -p pkg
 [ -e pkg/crx ] && rm -rf pkg/crx
 
-# Only generate the sqlite database if any rulesets have changed. Tried
+# Clean up obsolete ruleset databases, just in case they still exist.
+rm -f "$PWD/src/chrome/content/rules/default.rulesets" "$PWD/src/defaults/rulesets.sqlite"
```

Comment:
@jsha I used `-f` to stop it from throwing an error if the files already don't exist. But i could switch it to `|| true` or something; what do you think?

Good point about `$PWD`. It's also used when `RULESETS_JSON` is set, want to change that too?